package.json

```diff
@@ -1467,6 +1468,16 @@
                     }
                 }
             },
+            {
+                "command": "aws.samVisualize.renderTemplate",
+                "title": "%AWS.command.samVisualize.renderTemplate%",
```

Comment:
You'll need to add a corresponding value for `AWS.command.samVisualize.renderTemplate` in `package.nls.json` This is what the command will be called when users see it in the command palette.

Reply:
Yep, that's done, I just forgot to commit the `package.nls.json`.